status-im/status-react
# [#13263] Fix for ENS warning showing on custom domains

[comment]: # (Please replace ... with your information. Remove < and >)
[comment]: # (To auto-close issue on merge, please insert the related issue number after # i.e fixes #566)

### Summary

[comment]: # (Summarise the problem and how the pull request solves it)
Fixes #13263 

### Review notes


### Testing notes


#### Platforms


- Android
- iOS

### Steps to test
- Open Status
- Add an account that owns an custom (non *.stateofus.eth) ENS that does not have a ECDSA pubkey setup yet
- Verify that one can proceed through setup flow for ENS name in Settings


status: ready
